CONTRIBUTING.md

```diff
@@ -910,6 +911,24 @@ To debug your CDK application along with the CDK repository,
 6. The debug view, should now have a launch configuration called 'Debug hello-cdk' and launching that will start the debugger.
 7. Any time you modify the CDK app or any of the CDK modules, they need to be re-built and depending on the change the `link-all.sh` script from step#2, may need to be re-run. Only then, would VS code recognize the change and potentially the breakpoint.
 
+### Run a CDK unit test in the debugger
```

Comment:
Would be nice to rewrite the above section to use the same launch configuration and drop the step that asks the user to configure their own `launch` section?

Would help simplify the contribution guide, but I'll leave it to you to decide.